src/fileformat/file_format/pe/pe_format.cpp

```diff
@@ -833,7 +833,7 @@ void PeFormat::loadResourceNodes(std::vector<const PeLib::ResourceChild*> &nodes
 		return;
 	}
 
-	Resource resource;
+	std::unique_ptr<Resource> resource = std::make_unique<Resource>();
```

Comment:
You can use `auto` here. It is obvious from the initialization what is the type of the variable.